po/tr_TR.UTF-8.po

```diff
@@ -0,0 +1,270 @@
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-write.blp:6
+msgid "Authorize Clipboard Access"
+msgstr "Pano Erişimine İzin Ver"
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-read.blp:7
+msgid ""
+"An application is attempting to read from the clipboard. The current "
+"clipboard contents are shown below."
+msgstr ""
+"Bir uygulama panodan okumaya çalışıyor. Geçerli pano içeriği aşağıda "
+"gösterilmektedir."
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-read.blp:10
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-write.blp:10
+msgid "Deny"
+msgstr "Reddet"
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-read.blp:11
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-write.blp:11
+msgid "Allow"
+msgstr "İzin Ver"
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-write.blp:7
+msgid ""
+"An application is attempting to write to the clipboard. The current "
+"clipboard contents are shown below."
+msgstr ""
+"Bir uygulama panoya yazmaya çalışıyor. Geçerli pano içeriği aşağıda "
+"gösterilmektedir."
+
+#: src/apprt/gtk/ui/1.5/ccw-paste.blp:6
+msgid "Warning: Potentially Unsafe Paste"
+msgstr "Uyarı: Tehlikeli Olabilecek Yapıştırma"
+
+#: src/apprt/gtk/ui/1.5/ccw-paste.blp:7
+msgid ""
+"Pasting this text into the terminal may be dangerous as it looks like some "
+"commands may be executed."
+msgstr ""
+"Bu metni uçbirime yapıştırmak tehlikeli olabilir; çünkü bir komut "
+"yürütülecek gibi duruyor."
+
+#: src/apprt/gtk/inspector.zig:144
+msgid "Ghostty: Terminal Inspector"
+msgstr "Ghostty: Uçbirim Denetçisi"
+
+#: src/apprt/gtk/Surface.zig:1243
+msgid "Copied to clipboard"
+msgstr "Panoya kopyalandı"
+
+#: src/apprt/gtk/CloseDialog.zig:47
+msgid "Close"
+msgstr "Kapat"
+
+#: src/apprt/gtk/CloseDialog.zig:87
+msgid "Quit Ghostty?"
+msgstr "Ghostty’den Çık?"
+
+#: src/apprt/gtk/CloseDialog.zig:88
+msgid "Close Window?"
+msgstr "Pencereyi Kapat?"
+
+#: src/apprt/gtk/CloseDialog.zig:89
+msgid "Close Tab?"
+msgstr "Sekmeyi Kapat?"
+
+#: src/apprt/gtk/CloseDialog.zig:90
+msgid "Close Split?"
+msgstr "Bölmeyi Kapat?"
+
+#: src/apprt/gtk/CloseDialog.zig:96
+msgid "All terminal sessions will be terminated."
+msgstr "Tüm uçbirim oturumları sonlandırılacaktır."
+
+#: src/apprt/gtk/CloseDialog.zig:97
+msgid "All terminal sessions in this window will be terminated."
+msgstr "Bu penceredeki tüm uçbirim oturumları sonlandırılacaktır."
+
+#: src/apprt/gtk/CloseDialog.zig:98
+msgid "All terminal sessions in this tab will be terminated."
+msgstr "Bu sekmedeki tüm uçbirim oturumları sonlandırılacaktır."
+
+#: src/apprt/gtk/CloseDialog.zig:99
+msgid "The currently running process in this split will be terminated."
+msgstr "Bu bölmedeki şu anda çalışan süreç sonlandırılacaktır."
+
+#: src/apprt/gtk/Window.zig:200
+msgid "Main Menu"
+msgstr "Ana Menü"
+
+#: src/apprt/gtk/Window.zig:221
+msgid "View Open Tabs"
+msgstr "Açık Sekmeleri Görüntüle"
+
+#: src/apprt/gtk/Window.zig:295
+msgid ""
+"⚠️ You're running a debug build of Ghostty! Performance will be degraded."
+msgstr ""
+"⚠️ Ghostty’nin hata ayıklamaya özel bir sürümünü kullanıyorsunuz! Başarım "
```

Comment:
`Başarım` yerine `Performans` kullanılabilir.

Reply:
I think `Başarım` would also be okay here.